src/server/src/main/java/io/cassandrareaper/jmx/JmxProxyImpl.java

```diff
@@ -112,6 +111,7 @@
   private final String hostBeforeTranslation;
   private final JMXServiceURL jmxUrl;
   private final String clusterName;
+  private final HashMap<Integer, RepairStatusHandler> repairStatusHandlers = Maps.newHashMap();
```

Comment:
Seems like you can also remove the old `private final Optional<RepairStatusHandler> repairStatusHandler`.

Reply:
Right ! 

It's deleted now.